OPAL/br/src/main/scala/org/opalj/br/analyses/cg/EntryPointFinder.scala

```diff
@@ -347,4 +347,46 @@ object AllEntryPointsFinder extends EntryPointFinder {
             project.allProjectClassFiles.flatMap(_.methodsWithBody)
         else project.allMethodsWithBody
     }
+}
+
+/**
+ * The AndroidEntryPointFinder considers specific methods of app components as entry points.
+ * It does not work for androidx
+ *
+ * @author Tom Nikisch
+ */
+object AndroidEntryPointsFinder extends EntryPointFinder {
+
+    val activityEPS: List[String] = List("onCreate", "onRestart", "onStart", "onResume",
+        "onStop", "onDestroy", "onActivityResult")
+    val serviceEPS: List[String] = List("onCreate", "onStartCommand", "onBind", "onStart")
+    val contentProviderEPS: List[String] = List("onCreate", "query", "insert", "update")
+    val locationListenerEPS: List[String] = List("onLocationChanged", "onProviderDisabled", "onProviderEnabled",
+        "onStatusChanged")
+    val onNmeaMessageListenerEPS: List[String] = List("onNmeaMessage")
+    val broadcastReceiverEPS: List[String] = List("onReceive")
+    val defaultEPS = Map("android/app/Activity" -> activityEPS, "android/app/Service" -> serviceEPS,
```

Comment:
It seems it would be sensible to have this in a configuration file instead of hardcoded. You can easily represent such a Map[String, List[String]] in the config format and load that where you need it.